src/ansys/simai/core/utils/auth.py

```diff
@@ -46,15 +46,16 @@
 
 # polling can't be faster or auth server returns HTTP 400 Slow down
 DEVICE_AUTH_POLLING_INTERVAL = 5
+# Try to refresh tokens 300-400 secs before they go bad
+# - Randomized to prevent thundering herd
+# - Accounts for network latency and clock skew
+TOKEN_REFRESH_BUFFER = random.randrange(300, 400)  # noqa: S311
+TOKEN_EXPIRATION_BUFFER = random.randrange(5, 15)  # noqa: S311
```

Comment:
```suggestion
TOKEN_REFRESH_BUFFER = random.randrange(300, 400)  # noqa: S311 # nosec
TOKEN_EXPIRATION_BUFFER = random.randrange(5, 15)  # noqa: S311 # nosec
```
For `check-vulnerabilities` job

Reply:
Nice, thanks !